cmd/policy/README.md

```diff
@@ -0,0 +1,55 @@
+## Example usage
+
+NOTE: All commands allow specifying a tenant with -t. If this is not specified,
+then tenant 1 is usually assumed. The only exceptions are list command (which
+provides output for all tenants), and verify command (which will read tenant ID
+from the provided evidence context).
+
+    ./policy list
+
+List all policies in Veraison store. This will output a list of tenant ID and
+format name pairs.
+
+    ./policy -t 2 list
+
+Same as above but only list policies for tenant ID 2.
+
+    ./policy add -f my_policies.zip
```

Comment:
If I am not mistaken this should say `set` now?

Reply:
Do'h, missed that. Now fixed.